java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -1104,7 +1104,7 @@ void testWindowStatic() {
       }
 
       try (ColumnVector expected = ColumnVector.fromInts(2, 3, 3, 3, 2);
```

Comment:
sorry, but can you include nulls in this vector to really know if its counting valids only? 

can you also add a test for count_all?